CaptureTool/CaptureTool/PressedKeys.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Win32;
+using System.Windows.Forms;
+
+namespace CaptureTool
+{
+    class PressedKeys
+    {
+        private static List<int> currentPressedKeys = new List<int>();
```

Comment:
Recommended to initialize in the constructor

Reply:
Done